packages/yourloops/pages/hcp/team-member-remove-dialog.tsx

```diff
@@ -118,15 +118,16 @@ function RemoveMemberDialog(props: RemoveMemberDialogProps): JSX.Element {
         <Button
           id="team-members-dialog-rmmember-button-cancel"
           onClick={handleClose}
-          className={`${dialogClasses.buttonCancel} ${buttonClasses.buttonCancel}`}
-          variant="contained">
+          variant="contained"
```

Comment:
Removing the theme on all cancel buttons, change the way it was previously designed.
Was it intended in this Jira ?

Reply:
Working on consistency for the whole application, Mathieu agrees with these minor changes